infra/examples-dev/aws-msft-365/main.tf

```
@@ -175,15 +175,15 @@ module "psoxy-msft-connector" {
 
   source = "../../modules/aws-psoxy-instance"
 
-  function_name        = "psoxy-${each.key}"
-  source_kind          = each.value.source_kind
-  path_to_function_zip = module.psoxy-package.path_to_deployment_jar
-  function_zip_hash    = module.psoxy-package.deployment_package_hash
-  path_to_config       = "../../../configs/${each.value.source_kind}.yaml"
-  api_caller_role_arn  = module.psoxy-aws.api_caller_role_arn
+  function_name            = "psoxy-${each.key}"
+  source_kind              = each.value.source_kind
+  path_to_function_zip     = module.psoxy-package.path_to_deployment_jar
+  function_zip_hash        = module.psoxy-package.deployment_package_hash
+  path_to_config           = "../../../configs/${each.value.source_kind}.yaml"
```

should be here then too, right??
```suggestion
  path_to_config           = "${var.psoxy_basedir}/configs/${each.value.source_kind}.yaml"
```

---

Yes, I was hesitant, are these used?